### PR TITLE
Add API docs API.md

### PR DESCRIPTION
**Pull request by ticklemynausea**

### CODE REVIEW

File: API.md [hunk 1]
```diff
@@ -0,0 +1,204 @@
...
+
+## Locations
+
+### `GET /locations/countries`
+
+Lists all locations that are countries
+
+#### Query parameters
+
+* `topojson` - if present, the location's topojson structure will be sent with the object.
```
Comment (simaob) [inline API.md]: Should this be treated as a boolean? i.e.: `topojson=true`? Or it doesn't matter?

File: API.md [hunk 2]
```diff
@@ -0,0 +1,204 @@
...
+
+Collection is serialized as an object with the metadata entity as key
+
+### `GET /ndcs/:iso/sdgs`
+
+Shows linkage data for the given country code.
+
+### `GET /ndcs/sdgs_overview`
+
+Shows an overview of linkage data to meet specific front-end requirements:e .
```
Comment (simaob) [inline API.md]: there's a trailing `:e .` I guess it's a vim artefact =)

File: API.md [hunk 3]
```diff
@@ -0,0 +1,204 @@
...
+
+#### Query parameters
+
+* `location` - Filter by location iso code3. Accepts multiple values separated by commas.
+
+## Quantifications
+
+### `GET /quantifications`
+
+Lists quantification data
```
Comment (simaob) [inline API.md]: Suggest making it a bit more clear: "Lists quantification data, representing NDC emissions targets.

File: API.md (hunk 1 above)
Comment (ticklemynausea) [reply API.md]: It doesn't matter. The only thing required is that the parameter is present.

File: API.md (hunk 2 above)
Comment (ticklemynausea) [reply API.md]: Derp.

File: API.md (hunk 3 above)
Comment (ticklemynausea) [reply API.md]: :+1:

**Pull request merged**